### PR TITLE
Add Elasticsearch centralised logging option

### PR DESCRIPTION
**Pull request by jimmidyson**
FYI  @jwhonce: would you mind reviewing? I've added Elasticsearch centralised logging documentation in addition to the centralised filesystem that you originally added.

### CODE REVIEW

**Comment by jwhonce** [comment]
LGTM -- I like the breakdown.

File: admin_guide/aggregate_logging/overview.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+container logs to a centralized location. Automation can be found at
+https://github.com/openshift/openshift-ansible.
+
+IMPORTANT: This solution is a work in progress. As packaging improvements are
+made these instructions will be simplified.
+
+There are two options for aggregating logs, depending on user requirements:
+
+. link:aggregate_logging_filesystem.html[Storing logs on a centralised host's filesystem]
+. link:aggregate_logging_elasticsearch.html[Storing logs in Elasticsearch]
```
Comment (adellape) [inline admin_guide/aggregate_logging/overview.adoc]: Links should be `filesystem.html` and `elasticsearch.html` respectively.

**Comment by adellape** [comment]
@jimmidyson Do you feel strongly about having them in their own subdir as separate topics? Granted it wouldn't be the shortest topic ever, but it seems like this could all work as one.

I guess that gets into how we expect logging to evolve in the future, as maybe more options are included?

**Comment by jimmidyson** [comment]
@adellape Happy to take guidance on doc structure. Just didn't want to confuse storing logs on central filesystem as storing on ES. They're an either/or & felt it read better like that, but like I said happy to take advice on what works best for users.

**Comment by adellape** [comment]
@jimmidyson Yeah, was concerned about that possible either/or confusion as well. If in a single topic, maybe the Level 1 headings could be phrased something like "Using a Centralized Filesystem" and "Using Elasticsearch" to try to get that point across, with the "Overview" section at the top of the topic clearly stating the situation?

**Comment by jimmidyson** [comment]
@adellape Updated. What do you think? Reads OK to me.

**Comment by adellape** [comment]
@jimmidyson Shoot, sorry. Missed getting back to this. The combined topic LGTM, and I'd merge but there's a couple broken links based on the initial separate-topic layout (I'll ID them inline).

Two other thing that I could handle in a follow up, or you can take care of if you want:

1) Using the newer `oc` instead of `osc`.
2) Converting the JSON examples from v1beta3 to v1.

File: admin_guide/aggregate_logging.adoc [hunk 2]
```diff
@@ -7,17 +7,25 @@
...
+IMPORTANT: These solutions are a work in progress. As packaging improvements are
 made these instructions will be simplified.
 
+---
+
+= Using a Centralized Filesystem
+
+This option will read all container logs & forward them to a central server
+for storage on the filesystem. This solution requires less resources &
+require less management than the link:elasticsearch.html[Elasticsearch] option, but
```
Comment (adellape) [inline admin_guide/aggregate_logging.adoc]: Broken link.

File: admin_guide/aggregate_logging.adoc [hunk 3]
```diff
@@ -172,3 +180,261 @@ TIP: Any errors will be logged in the *_/var/log/td-agent/td-agent.log_* file.
...
+= Using Elasticsearch
+
+https://www.elastic.co/products/elasticsearch[Elasticsearch] is an open source
+distributed document database that indexes documents & provides full-text
+search capabilities. By storing container logs in Elasticsearch, users will be
+able to search all content & filter appropriately. This documentation shows how
+to run https://www.elastic.co/products/kibana[Kibana].
+
+This option requires more configuration & more resources than the
+link:filesystem.html[centralised filesystem] option, but makes logs more useful
```
Comment (adellape) [inline admin_guide/aggregate_logging.adoc]: Broken link.

**Comment by jimmidyson** [comment]
@adellape Thanks. I'll sort out asap

**Comment by jimmidyson** [comment]
@adellape Updated.

**Comment by adellape** [comment]
@jimmidyson Awesome, thanks. Merging.

**Comment by jimmidyson** [comment]
Thanks @adellape!

**Pull request merged**